.circleci/config.yml

```diff
@@ -281,6 +281,25 @@ memcached: &memcached
   - image: memcached:1.6.9
 
 jobs:
+  "node-18":
+    docker:
+      - image: circleci/node:18
```

Comment:
@kirrg001 My guess is that circleci/node:18 might never become available. See https://circleci.com/blog/announcing-our-next-generation-convenience-images-smaller-faster-more-deterministic/ - they recently switched from circleci/$image-name to cimg/$image-name. `cimg/node:18` might already be available according to https://circleci.com/developer/images/image/cimg/node.

The most recent image for the circleci namespace on Dockerhub is 4 month old (Node 16.13), so they apparently stopped publishing new `circleci/*` images.

We could try with `cimg/node:18` here. If that works, we could also try switching the other Node.js versions to the `cimg` namespace, depending on whether that is a drop-in replacement or introduces additional effort.

Reply:
Ugh, just tested it, looks like they do not tag with only the major version. That is, cimg/node:18 does not exist, but cimg/node:18.0 exists - we need to specify the minor version also. Not exactly great for our use case, since we then need to remember to switch to more recent minor versions once they become available :-/